src/common/src/ConfirmDialog/ConfirmDialog.test.tsx

```diff
@@ -51,4 +54,35 @@ describe('<ConfirmDialog/>', () => {
 		userEvent.click(confirmButton);
 		expect(onConfirmMock).toBeCalled();
 	});
+
+	it('should call the onClose function when clicking the closeButton', () => {
+		const confirmButton = getByTestId('modal-close-button');
+		userEvent.click(confirmButton);
+		expect(onCloseMock).toBeCalled();
+	});
 });
+
+describe('<ConfirmDialog />', () => {
+	test('Should not show close button',  () => {
+		const props = {
+			title: 'TestDialog',
+			message: 'Are you sure you want to delete this file?',
+			textCancelButton: 'Cancel',
+			textConfirmButton: 'Confirm',
+			onCancel: jest.fn(),
+			onConfirm: jest.fn(),
+			onCloseButton: jest.fn()
+		} as IState;
+
+		render(
+			<div>
+				<button data-testid="open-dialog" onClick={() => confirm(props)} />
+				<ConfirmDialog />
+			</div>);
+
+		const button = screen.getByTestId('open-dialog');
+		userEvent.click(button);
+
+		expect(screen.queryByTestId('modal-close-button')).toBeNull();
```

Comment:
`.toBeInTheDocument()` woudl be a better matcher to use in this case.

Reply:
When using `not.toBeInTheDocument()`, the test fails and returns: `TestingLibraryElementError: Unable to find an element by: [data-testid="modal-close-button"]`. For this reason I use the `queryByTestId` method instead of `getByTestId`. Also, the close button doesn't contain any text, so `getByText` is not an option either unfortunately.

Reply:
Updated